recipes/kraken-all/meta.yaml

```diff
@@ -0,0 +1,13 @@
+package:
+  name: kraken-all
+  version: "0.10.5beta"
+requirements:
+  build:
+    - kraken ==0.10.5beta
```

Comment:
So it seems that when it is installed, it will reuse the version that is already there hence they are implicitly linked?